gradle/libs.versions.toml

```diff
@@ -12,8 +12,14 @@ androidXTest = "1.5.0"
 testOrchestrator = "1.4.2"
 lifecycle = "2.5.1"
 thirdPartyReport = "0.17.798"
+kaspressoSnapshotVersion = "1.5.1-SNAPSHOT"
```

Comment:
@Nikitae57 maybe we should set snapshot version through input fields like here https://github.com/KasperskyLab/Kaspresso/blob/master/.github/workflows/publication_maven.yml. What do you think?

Reply:
if this new action will be triggered by manual workflow run, than we will not need to set needed version in another commit before run